quint/src/cli.ts

```diff
@@ -248,6 +253,20 @@ const docsCmd = {
   handler: (args: any) => load(args).then(chainCmd(docs)).then(outputResult),
 }
 
+const validate = (argv: any) => {
+  if (argv.output && typeof argv.output === 'string') {
+    const output = argv.output
+    if (!output.endsWith('.itf.json')) {
+      throw new Error(`Unexpected format in --output: ${output}`)
+    }
+    if (!output.includes('{}') && !output.includes('{#}')) {
```

Comment:
Shouldn't we always require `{}`?

Reply:
if there is no `{}` and there is no `{#}`, the arguments parser will throw an error. I am assuming that both `{}` and `{#}` are unique.

Reply:
I can imagine a case, where test names are less relevant than their indices. For example, when we just want to process all of them.